sdk/attestation/azure-security-attestation/README.md

```diff
@@ -0,0 +1,180 @@
+
+Attestation is a process for demonstrating that software binaries were properly instantiated on a trusted platform. Remote relying parties can then gain confidence that only such intended software is running on trusted hardware. Azure Attestation is a unified customer-facing service and framework for attestation.
+
+Azure Attestation enables cutting-edge security paradigms such as Azure Confidential computing and Intelligent Edge protection. Customers have been requesting the ability to independently verify the location of a machine, the posture of a virtual machine (VM) on that machine, and the environment within which enclaves are running on that VM. Azure Attestation will empower these and many additional customer requests.
+
+Azure Attestation receives evidence from compute entities, turns them into a set of claims, validates them against configurable policies, and produces cryptographic proofs for claims-based applications (for example, relying parties and auditing authorities).
+
+> NOTE: This is a preliminary SDK for the Microsoft Azure Attestation service. It provides all the essential functionality to access the Azure Attestation service, but requires a significant amount of infrastructure to work correctly.
+
+
+## Getting started
+### Adding the package to your project
+Maven dependency for the Azure Attestation  library. Add it to your project's POM file.
+
+[//]: # ({x-version-update-start;com.azure:azure-security-attestation;current})
+```xml
+<!-- Install the Azure Attestation SDK -->
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-security-attestation</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Prerequisites
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- An existing [Azure Attestation][azure_attestation]. If you need to create an attestation instance, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-instance-name>` with your own, unique names:
+
+    ```bash
+    az attestation create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+    ```
+
+### Authenticate the client
+In order to interact with the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
+
+The simplest way of providing a bearer token is to use the  `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+
+
+## Key concepts
+
```

Comment:
Should `Key  Concept` talk about different kind of client you have for this SDK ?

Reply:
Isn't that what is in the "There are for major components in the preview SDK" section? In my mind, that describes the client types.

Would you rather I change the word "components" to "client types"?

Please note that this is completely going to change in the convenience layer when I write that - all the existing client types are likely to be consolidated into a single client type (possibly into two client types, but most likely a single client type).